ui/mirage/scenarios/default.js

```diff
@@ -207,17 +207,17 @@ function variableTestCluster(server) {
 
   server.create('variable', {
     id: `nomad/jobs/${variableLinkedJob.id}/${variableLinkedGroup.name}/${variableLinkedTask.name}`,
```

Comment:
`question`: why are these `id` properties being updated to include `namespace?

Reply:
@ChaiWithJai For demo purposes, we want the variables that link to jobs, to also have the same namespace as the jobs/groups/tasks they're linking to. Otherwise we'd have a 404ing link situation (which can arise! We don't stop you from making a variable that starts with `nomad/jobs/` against a job that does not exist). We just want our mirage demo flow to be on the happy path.